relay-ourlogs/src/otel_to_sentry.rs

```diff
@@ -62,9 +62,18 @@ pub fn otel_to_sentry_log(
     } = otel_log;
 
     let span_id = SpanId::try_from(span_id.as_slice())
-        .map_or_else(|err| Annotated::from_error(err, None), Annotated::new);
-    let trace_id = TraceId::try_from(trace_id.as_slice())
-        .map_or_else(|err| Annotated::from_error(err, None), Annotated::new);
+        .map(Annotated::new)
+        .unwrap_or_default();
```

Comment:
Mh, maybe we should have `try_from` return a better error, differentiate between empty and invalid, then we could annotate an error and ignore empty. But I guess for now assuming empty is fine.

Reply:
The `span_id` on `OurLog` is *not* required, so we could just set it to `Annotated::empty()` in that case, otherwise try to parse and if parsing fails, add the error?

Reply:
Yup, that looks like what I had in mind and it should work correctly, no span id attribute if it is missing.

From `processing/logs/store.rs`:
```rs
    if let Some(span_id) = span_id {
        result.insert(
            "sentry.span_id".to_owned(),
            AnyValue {
                value: Some(any_value::Value::StringValue(span_id.to_string())),
            },
        );
    }
```